spec/search-api.yaml

```diff
@@ -46,25 +46,48 @@ paths:
               schema:
                 $ref: "#/components/schemas/Table"
         '404':
-          description: The requested dataset is not available
+          description: This server does not implement the data operation
         '500':
           description: An unexpected error occurred
   /table/{table_name}/data:
     get:
       summary: Fetch data from a Table
       description: Optional operation that returns data from a Table
-      operationId: data
+      operationId: getData
+      parameters:
+        - name: table_name
+          in: path
+          description: Unique Table name, unique to this implementation
```

Comment:
Again, somewhat fuzzy. Maybe in this case, `Name of the table to fetch data from`